ci-operator/step-registry/hypershift/mce/kubevirt/create/disconnected/hypershift-mce-kubevirt-create-disconnected-chain.yaml

```diff
@@ -0,0 +1,211 @@
+            import.open-cluster-management.io/klusterlet-deploy-mode: Hosted
+            open-cluster-management/created-via: other
+          labels:
+            cloud: auto-detect
+            cluster.open-cluster-management.io/clusterset: default
+            name: \${CLUSTER_NAME}
+            vendor: OpenShift
+          name: \${CLUSTER_NAME}
+        spec:
+          hubAcceptsClient: true
+          leaseDurationSeconds: 60
+        EOF2
+
+        echo "Waiting for cluster to become available"
+        oc wait --timeout=30m --for=condition=Available --namespace=local-cluster hostedcluster/\${CLUSTER_NAME}
+        echo "Cluster became available, creating kubeconfig"
+        /tmp/hcp create kubeconfig --namespace=local-cluster --name=\${CLUSTER_NAME} > /home/nested_kubeconfig
+
+        EOF
+
+        scp "${SSHOPTS[@]}" "root@${IP}:/home/nested_kubeconfig" "${SHARED_DIR}/nested_kubeconfig"
+
+        if [ "$(oc get infrastructure cluster -o=jsonpath='{.status.platformStatus.type}')" == "BareMetal" ]; then
+          exit 0
+        fi 
+
+        echo "Waiting for nested cluster's node count to reach the desired replicas count in the NodePool"
+        until \
+          [[ $(oc get nodepool ${CLUSTER_NAME} -n local-cluster -o jsonpath='{.spec.replicas}') \
+            == $(oc --kubeconfig=${SHARED_DIR}/nested_kubeconfig get nodes --no-headers | wc -l) ]]; do
+              echo "$(date --rfc-3339=seconds) Nested cluster's node count is not equal to the desired replicas in the NodePool. Retrying in 30 seconds."
+              oc get vmi -n ${CLUSTER_NAMESPACE}
+              sleep 30s
+        done
+
+        echo "Waiting for clusteroperators to be ready"
+        export KUBECONFIG=${SHARED_DIR}/nested_kubeconfig
+
+        until \
+        oc wait clusterversion/version --for='condition=Available=True' > /dev/null;  do
+          echo "$(date --rfc-3339=seconds) Cluster Operators not yet ready"
+          oc get clusteroperators 2>/dev/null || true
+          sleep 1s
+        done
+
+        echo "Waiting for ManagedCluster to be ready"
+        export KUBECONFIG=${SHARED_DIR}/kubeconfig
+        until \
+        oc wait managedcluster ${CLUSTER_NAME} --for='condition=ManagedClusterJoined' >/dev/null && \
+        oc wait managedcluster ${CLUSTER_NAME} --for='condition=ManagedClusterConditionAvailable' >/dev/null && \
+        oc wait managedcluster ${CLUSTER_NAME} --for='condition=HubAcceptedManagedCluster' >/dev/null;  do
+        echo "$(date --rfc-3339=seconds) ManagedCluster not yet ready"
+        sleep 10s
+        done
+
+      from_image:
+        namespace: ocp
+        name: "4.13"
+        tag: upi-installer
+      grace_period: 5m0s
+      resources:
+        requests:
+          cpu: 100m
+          memory: 100Mi
+      timeout: 50m0s
+      credentials:
+        - mount_path: /etc/ci-pull-credentials
+          name: ci-pull-credentials
+          namespace: test-credentials
```

Comment:
We not use credentials, so we can move it

Reply:
done